fe/fe-core/src/main/java/com/starrocks/connector/hive/glue/converters/CatalogToHiveConverter.java

```diff
@@ -152,7 +154,13 @@ public static Table convertTable(com.amazonaws.services.glue.model.Table catalog
         Date lastAccessedTime = catalogTable.getLastAccessTime();
         hiveTable.setLastAccessTime(lastAccessedTime == null ? 0 : (int) (lastAccessedTime.getTime() / 1000));
         hiveTable.setRetention(catalogTable.getRetention());
-        hiveTable.setSd(convertStorageDescriptor(catalogTable.getStorageDescriptor()));
+        // for iceberg table, don't need to set StorageDescriptor
+        // just use metadata location in parameters that checked in HiveTableValidator
+        // TODO(zombee0), check hudi deltalake
+        if (catalogTable.getParameters() == null || catalogTable.getParameters().get(TABLE_TYPE_PROP) == null ||
```

Comment:
It's better to abstract a method named isIcebergTable()

Reply:
done